src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -209,14 +209,15 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
               )}
 
               {shouldShowSchedule ? (
-                <div
+                <button
+                  aria-label="Schedule"
```

Comment:
Should the label not be `navbarTitle` and "Schedule" given that's what the visible text is?

Reply:
I believe they are also going to read the default text on the button though. I’m pretty sure this attribute only needs to be added when the standard text doesn’t exist/work (eg. When it’s an icon only button)

Reply:
> I believe they are also going to read the default text on the button though. I’m pretty sure this attribute only needs to be added when the standard text doesn’t exist/work (eg. When it’s an icon only button

@oleksii-front have you checked if it's true?

Reply:
@mike-lvov It's true. But in this case the content is whole 'events' dialog. This change forces to read this button right.